sdk/monitor/Azure.Monitor.Query/src/MetricsQueryClient.cs

```diff
@@ -16,17 +16,30 @@ namespace Azure.Monitor.Query
     /// </summary>
     public class MetricsQueryClient
     {
+        private static readonly Uri _defaultEndpoint = new Uri("https://management.azure.com");
+
         private readonly MetricDefinitionsRestClient _metricDefinitionsClient;
         private readonly MetricsRestClient _metricsRestClient;
         private readonly MetricNamespacesRestClient _namespacesRestClient;
         private readonly ClientDiagnostics _clientDiagnostics;
 
         /// <summary>
-        /// Initializes a new instance of <see cref="MetricsQueryClient"/>.
+        /// Initializes a new instance of <see cref="MetricsQueryClient"/> using the default endpoint.
```

Comment:
I think this text should be similar to what we use for the `LogsQueryClient` constructor. With that in mind, here's my suggestion:

```suggestion
        /// Initializes a new instance of <see cref="MetricsQueryClient"/>. Uses the default 'https://management.azure.com' endpoint.
```